src/components/grapher/GrapherDots.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <g class="grapher-dots--dots-container">
+    <circle
+      v-for="dot in stuntSheetDots"
+      :key="`${dot.x}-${dot.y}-dots--dot`"
+      class="grapher-dots--dot"
+      :cx="dot.xAtBeat(beat - 1)"
+      :cy="dot.yAtBeat(beat - 1)"
+      r="0.7"
+      data-test="grapher-dots--dot"
+    />
+    <g v-if="showDotLabels">
+      <text
+        v-for="(dot, index) in stuntSheetDots"
+        :key="`${dot.x}-${dot.y}-dots--dottext`"
+        class="grapher-dots--dottext"
+        :x="dot.xAtBeat(beat - 1)"
+        :y="dot.yAtBeat(beat - 1) - 1"
+        data-test="grapher-dots--dottext"
+      >
+        {{ dotLabels[index] }}
+      </text>
+    </g>
```

Comment:
We can actually use a `v-for` that wraps around the dot and the text! For example...

```vue
<g v-for="(dot, index) in stuntsheetDots"/>
  <circle :cx="dot.xAtBeat(beat - 1)"
    ...
  />
  <text v-if="showDotLabels"
    :x="dot.xAtBeat(beat-1)"
    ...
  />
    {{ dotLabels[index] }}
  </text>
</g>
```

Reply:
Actually, I can't do that:

```
  Errors compiling template:

  Cannot use v-for on stateful component root element because it renders multiple elements.

  1  |  
  2  |  <g v-for="dot in stuntSheetDots"
     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  3  |    class="grapher-dots--dots-container">
  4  |    <circle
```

This is because the <g> section has multiple elements, so you can't have a for loop there.  So that's why I did it as two elements.

Unless there's a different way to do this that I don't know about.